PROJECT

```diff
@@ -0,0 +1,16 @@
+domain: openstack.org
+layout: go.kubebuilder.io/v2
+repo: github.com/openstack-k8s-operators/neutron-operator
+resources:
+- group: neutron
+  kind: OVSNodeOsp
+  version: v1beta1
```

Comment:
this requires a change to the compute-node-operator bindata. I did the same to nova-operator as it reflects better the status of the CRD